tools/PI/DevHome.PI/PIApp.xaml

```diff
@@ -27,6 +27,7 @@
             </Style>
             <StackLayout x:Name="ExternalToolsHorizontalLayout" Orientation="Horizontal" />
             <StackLayout x:Name="ExternalToolsVerticalLayout" Orientation="Vertical" />
+

```

Comment:
NIT: extra line.

Reply:
Yes, leaving blank lines between different functional definitions seems reasonable, no?

Reply:
Sorry, I meant "functional" not "function" - different definitions in the XAML that serve different functional purposes. Anyway, I've added another blank after the </Style> closing tag, as you suggested.